asapdiscovery-alchemy/asapdiscovery/alchemy/cli/alchemy.py

```diff
@@ -108,13 +108,6 @@ def plan(
             for mol in alchemy_ds.posed_ligands
         ]
 
-        # if any ligands lack a name, then raise an exception; important for
```

Comment:
FWIW, might still be a good idea to add a user-warning when we're adding names in the background. This might prompt the user to cancel and sort out the ligand names instead.

Reply:
I'm not sure what you mean by this. Do we want a warning that allows no-names through, or do we want to stop them from happening at all?

Reply:
NVM - I was under the impression we'd be setting ligand names with this PR, rather than what you've built here. IMO what you've made here (i.e. forcing the user to go back and set ligand names) is a better approach, so can disregard my above comment.